netgrasp/utils/mac.py

```diff
@@ -0,0 +1,255 @@
+from __future__ import print_function
+
+import os
+import re
+import datetime
+
+TEST_WSDATA = False
+TEST_WSDATA_FILENAME = '~/.netgrasp.mac.wsdata'
```

Comment:
What's being cached to the filesystem? Why? This runs the risk of causing permissions issues when trying to use netgrasp.